src/lib/openapi/spec/legal-value-schema.ts

```diff
@@ -4,13 +4,19 @@ export const legalValueSchema = {
     $id: '#/components/schemas/legalValueSchema',
     type: 'object',
     additionalProperties: false,
+    description:
+        'Describes a legal value. Typically used to limit possible values for contextFields or strategy properties',
     required: ['value'],
     properties: {
         value: {
+            description: 'The valid value',
             type: 'string',
+            example: 'red',
         },
         description: {
+            description: 'Describes the intended usage of this value',
             type: 'string',
+            example: 'We only carry these colors in our shop',
```

Comment:
This description's example feels like it's for legal values as a whole and not this specific value. How about something like this (if we're sticking with colors)?


```suggestion
            description: 'The valid value',
            type: 'string',
            example: '#c154c1',
        },
        description: {
            description: 'Describes this specific legal value',
            type: 'string',
            example: 'Deep fuchsia'
```

Reply:
Yeah, that works for me :) Thanks for the improved example and description